src/brackets/index.js

```diff
@@ -4,6 +4,25 @@
  * @param {string} str The string of brackets.
  * @returns {"valid" | "invalid"} Whether or not the string is valid.
  */
-function isValid(str) {}
+function isValid(str) {
+   if(str.length%2 > 0){
+        return "invalid";
+    }
+    var check ={ ")":"(", "]":"[", "}":"{" };
```

Comment:
Why is this reversed?

Reply:
It can still work if reversed, but the code will be adjusted a bit. 
Thanks for the review...